src/markdown-pages/contribute-to-quickstarts/instrument-library/index.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: 'Instrument your Library'
+template: 'OverviewTemplate'
+description: ''
+---
+<Intro>
+If your looking to jump into building your own quickstart, the first step is instrumenting your own library to send data to New Relic. New Relic I/O gives end users the ability to slice and dice observability data around their application, server, etc. Setting up your library to get data into New Relic is critical to this process, so that the end users have a streamlined click-thorugh experience to see your data on our platform. You will need to instrument your library with at least one data type to be able to observe it via a dashboard or alert.
+</Intro>
+
+## Observability Data Options
+
+ New Relic has [four types of observability data](https://docs.newrelic.com/docs/data-apis/understand-data/new-relic-data-types) which we call MELT data; *Metrics, Events, Logs, and Traces*. Each one handles a specific use case, so you'll have to decide which one fits your needs best.
+
+ Here is some context to help you decide:
+
+ - [Events](contribute-to-quickstarts/create-a-quickstart/instrument-library/send-events) **(Most Common)**: Events are used if you are capturing something occuring in your application, server, library, etc. For CI/CD applications this might be a capture of each time a job runs. For security applications this might be any time a threat is detected.
+ - [Logs](https://docs.newrelic.com/docs/logs/log-api/introduction-log-api/) **(Common)**: Logs should be used if your library uses multiple attributes (ie. nested key-value data), or if your library is set up already to use log drains.
+ - [Metrics](https://docs.newrelic.com/docs/data-apis/ingest-apis/metric-api/introduction-metric-api/) **(Uncommon)**: Metrics are generally collected as aggregated data or as a numeric status at a point in time. Most metrics can be sent or aggregated through events, which is why we recommend events instead. You might want to use metrics if you view the data at a very high level, and don't expect your users to need a more detailed analysis.
```

Comment:
Do you need to specify which of these are common and uncommon? Can we just let the partner decide which ones would be valuable for them?

Reply:
I don't have to, I thought it might be a way to point them towards events or logs which is really what makes the most sense. I think if the descriptions are thorough enough that should be clear though.